inference-engine/src/mkldnn_plugin/nodes/subgraph.cpp

```diff
@@ -0,0 +1,861 @@
+    parallel_for5d(dom[0], dom[1], dom[2], dom[3], dom[4],
+        [&](size_t d0, size_t d1, size_t d2, size_t d3, size_t d4) {
+            CallArgs ca;
+            // Benchmarking for overhead
+            for (size_t i = 0; i < n; i++) {
+                ca.push(inputs[i] + d0*offsets_in[i][0] + d1*offsets_in[i][1] + d2*offsets_in[i][2] + d3*offsets_in[i][3] + d4*offsets_in[i][4]);
+            }
+            for (size_t i = 0; i < m; i++) {
+                ca.push(outputs[i] + d0*offsets_out[i][0] + d1*offsets_out[i][1] + d2*offsets_out[i][2] + d3*offsets_out[i][3] + d4*offsets_out[i][4]);
+            }
+
+            schedule.get_callable<kernel>()(ca.raw(), sch.raw(), off.raw());
+        });
+}
+
+void MKLDNNSnippetNode::shedule_nt(const std::vector<uint8_t *>& outputs, const std::vector<const uint8_t *>& inputs) const {
+    size_t inputNum = inputs.size();
+    size_t outputsNum = outputs.size();
+
+    IE_THROW() << "Fix shedule_nt for a new calling convention";
+
+    parallel_nt(0, [&](const int ithr, const int nthr) {
+        size_t start = 0, end = 0;
+        splitter(schedulerWorkAmount, nthr, ithr, start, end);
+
+        auto work_size = dims_out[max_rank_out_desc_idx];
+        std::vector<size_t> counters(work_size.size(), 0);
+
+        for (size_t iwork = start; iwork < end; ++iwork) {
+            size_t tmp = iwork;
+            for (ptrdiff_t j = work_size.size() - 2; j >= 0; j--) {
+                counters[j] = tmp % work_size[j];
+                tmp /= work_size[j];
+            }
+
+            size_t index_in[7] = {0};
+            for (int i = 0; i < inputNum; i++) {
+                index_in[i] = 0;
+                for (int j = 0; j < counters.size(); j++) {
+                    index_in[i] += counters[j] * offsets_in[i][j];
+                }
+            }
+
+            size_t index_out[7] = {0};
+            for (int i = 0; i < outputsNum; i++) {
+                for (int j = 0; j < counters.size(); j++) {
+                    index_out[i] += counters[j] * offsets_out[i][j];
+                }
+            }
+
+            union param {
+                const float* ptr;
+                size_t len;
+            };
+
+            std::array<param, 8> args;
+
+            for (size_t i = 0; i < inputNum; i++) {
+                args[i].ptr = reinterpret_cast<const float*>(inputs[i] + index_in[i]);
+            }
+
+            for (size_t i = 0; i < outputsNum; i++) {
+                args[inputNum+i].ptr = reinterpret_cast<const float*>(outputs[i] + index_out[i]);
+            }
+
+            args[inputNum+outputsNum].len = static_cast<size_t>(work_size[work_size.size() - 1]);
+
+            typedef void (*ker)(const void *);
+            ker k = reinterpret_cast<ker>(const_cast<unsigned char*>(schedule.ptr));
+            k(&args[0]);
+        }
+    });
+}
+
+bool MKLDNNSnippetNode::evaluate(const ngraph::HostTensorVector& outputs, const ngraph::HostTensorVector& inputs) const {
```

Comment:
It seems that we don't, so it was removed by @v-Golubev in 8d7ce14791e3a1e9c38f5828793c2cf7efe2302c ([§] Review fixes)

Reply:
Concerning MKLDNNSnippetNode::shedule_nt (I mean this comment : https://github.com/openvinotoolkit/openvino/pull/7936#discussion_r736386389), I'm going to remove it since it's not used, and create a follow-up task.